src/main/resources/mediator.properties

```diff
@@ -1,11 +1,18 @@
-# Mediator Properties
+# == Mediator Properties == #
 mediator.name=HDR-mediator
 mediator.host=localhost
 mediator.port=3000
 mediator.timeout=60000
 mediator.heartbeats=true
 
-core.host=localhost
+# == OpenHIM Core Properties == #
+core.host=139.162.143.150
 core.api.port=8080
 core.api.user=root@openhim.org
```

Comment:
@cozej4 we should change these credentials make a separate user for the mediators, even though our OpenHIM instance is for dev/staging, it's still an internet facing service

Reply:
@nityan Noted. 
Any proposed vague sample credentials that we should have publicly available on this file?
I was afraid of adding the actual credentials since this is a public repo

Reply:
adding my response here from skype so it's tracked
`I think localhost is fine and admin and password should also be fine`